paper/paper.md

```diff
@@ -0,0 +1,434 @@
+---
+title: 'Teaching Python for Data Science: Collaborative development of a modular & interactive curriculum'
+tags:
+  - Data Science
+  - Python
+  - Jupyter
+  - Girls Who Code
+authors:
+  - name: Marlena Duda
+    orcid: TODO
+    affiliation: 1
+  - name: Kelly L. Sovacool
+    orcid: 0000-0003-3283-829X
+    affiliation: 1
+  - name: Negar Farzaneh
+    orcid: TODO
+    affiliation: 1
+  - name: Vy Nguyen
+    orcid: 0000-0002-6128-0523
+    affiliation: "1, 8"
+  - name: Sarah Haynes
+    orcid: 0000-0003-3225-1691
+    affiliation: 6
+  - name: Hayley Falk
+    orcid: 0000-0002-5198-1836
+    affiliation: 1
+  - name: Katherine L. Furman
+    orcid: 0000-0002-0340-4292
+    affiliation: "3, 4"
+  - name: Logan A. Walker
+    orcid: 0000-0002-5378-3315
+    affiliation: "5, 1"
+  - name: Rucheng Diao
+    orcid: TODO
+    affiliation: 1
+  - name: Morgan Oneka
+    orcid: TODO
+    affiliation: 1
+  - name: Audrey Drotos
+    orcid: TODO
+    affiliation: TODO
+  - name: Alana Woloshin
+    orcid: 0000-0002-5224-4221
+    affiliation: 7
+  - name: Gabrielle Dotson
+    orcid: TODO
+    affiliation: 1
+  - name: April Kriebel
+    orcid: 0000-0003-0008-9044
+    affiliation: 1
+  - name: Lucy Meng
+    orcid: TODO
+    affiliation: TODO
+  - name: Stephanie N. Thiede
+    orcid: 0000-0003-0173-4324
+    affiliation: 2
+  - name: Shweta Ramdas
+    orcid: TODO
+    affiliation: TODO
+  - name: Zena Lapp^[corresponding author]
+    orcid: 0000-0003-4674-2176
+    affiliation: 1
+  - name: Brooke N. Wolford^[corresponding author]
+    orcid: 0000-0003-3153-1552
+    affiliation: 1
+affiliations:
+  - name: Department of Computational Medicine & Bioinformatics, University of Michigan
+    index: 1
+  - name: Department of Microbiology & Immunology, University of Michigan
+    index: 2
+  - name: Neuroscience Graduate Program, University of Michigan
+    index: 3
+  - name: Michigan Neuroscience Institute, University of Michigan
+    index: 4
+  - name: Biophysics Graduate Program, University of Michigan
+    index: 5
+  - name: Department of Pathology, University of Michigan
+    index: 6
+  - name: School of Information, University of Michigan
+    index: 7
+  - name: Department of Environmental Health Sciences, University of Michigan
+    index: 8
+date: 30 Apr. 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+<!-- pasted from our PyCon 2020 talk proposal -->
+We are Bioinformatics PhD students at the University of Michigan who started a
```

Comment:
Should we still say this even though we're not all PhD students and not all in Bioinf? Although most of us are, including the first & corresponding authors.